openedx/core/djangoapps/profile_images/images.py

```diff
@@ -33,42 +36,32 @@
 }
 
 
-def user_friendly_size(size):
+def create_profile_images(image_file, profile_image_names):
     """
-    Convert size in bytes to user friendly size.
-
-    Arguments:
-        size (int): size in bytes
-
-    Returns:
-        user friendly size
+    Generates a set of image files based on image_file and
+    stores them according to the sizes and filenames specified
+    in `profile_image_names`.
```

Comment:
Could you add `Arguments` docstrings? It's not immediately clear to me what type/format I should be expecting for either argument here.